geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/StartServerCommand.java

```diff
@@ -202,9 +202,14 @@ public Result startServer(
 
     cacheXmlPathname = CliUtil.resolvePathname(cacheXmlPathname);
 
-    if (StringUtils.isNotBlank(cacheXmlPathname) && !IOUtils.isExistingPathname(cacheXmlPathname)) {
-      return ResultBuilder.createUserErrorResult(
-          CliStrings.format(CliStrings.CACHE_XML_NOT_FOUND_MESSAGE, cacheXmlPathname));
+    if (StringUtils.isNotBlank(cacheXmlPathname)) {
+      if (!IOUtils.isExistingPathname(cacheXmlPathname)) {
+        return ResultBuilder.createUserErrorResult(
+            CliStrings.format(CliStrings.CACHE_XML_NOT_FOUND_MESSAGE, cacheXmlPathname));
+      } else {
+        getGfsh().logWarning(
```

Comment:
Can you tell in this context if cluster config is enabled? If you can then you could test that and change the warning message.

Reply:
@dschneider-pivotal the rest of the string fixes are checked in. I spoke with gfsh developers and confirmed that since create server is not a connected command it cannot figure out if the cluster config is enabled.
If you think that these changes are enough please do approve the PR